src/usr/local/www/classes/Form/Checkbox.class.php

```diff
@@ -56,9 +56,9 @@ protected function _getInput()
 	{
 		$input = parent::_getInput();
 
-		if (empty($this->_description))
-			return '<label class="chkboxlbl">'. $input .'</label>';
+		if (!empty($this->_description) || is_numeric($this->_description))
```

Comment:
This fixes the case where the checkbox description is "0". Prior to this change, the description "0" was not displayed.
(Don't you love the PHP empty() function?!)